homeassistant/components/google_assistant/trait.py

```diff
@@ -317,7 +317,9 @@ def query_attributes(self):
         response = {}
 
         temp = self.state.attributes.get(light.ATTR_COLOR_TEMP)
-        if temp is not None:
+        # not checking specifically for `None` because some faulty
+        # integration might put 0 in here, raising exception.
+        if temp:
```

Comment:
If the integration is faulty, is this not a case of hiding the symptom?

Reply:
It is. I guess it would be better to bail out and notify Google light is not available.

Reply:
I've added a warning.